frontend/libs/studio-components/src/components/StudioAlert/StudioAlert.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import type { StudioAlertProps } from './StudioAlert';
+import { StudioAlert } from './StudioAlert';
+
+describe('StudioAlert', () => {
+  it('should render the component', () => {
+    renderTestAlert({ severity: 'danger' });
+    const studioAlert = screen.getByText('Feil');
```

Comment:
I'm not a huge fan of this test, but i did not find a better way to test anything of this component.
Suggestions for other ways to test this component is very welcome.

Reply:
I think we should use this utilities as well: https://github.com/Altinn/altinn-studio/tree/main/frontend/libs/studio-components/src/test-utils. To test fowrardRef, className etc. 😊

Reply:
Nice! Added a few tests using these